src/css/base.scss

```diff
@@ -1,76 +1,77 @@
 @import 'normalize';
-@import 'variables';
-@import 'media-queries';
+@import '_variables';
+@import '_media-queries';
 // @import 'mixins';
 
 * {
   box-sizing: border-box;
   margin: 0;
   padding: 0;
+  // border: 3px solid red;
 }
 
 %flex_center-shared {
   align-items: center;
   display: flex;
+  flex-direction: column;
+  justify-content: center;
 }
 
 body {
   @extend %flex_center-shared;
-  background-color: $background_color;
-  flex-direction: column;
+  background-color: $dark_main;
   font-family: 'Noto Sans SC', sans-serif;
+  height: 100vh;
 }
 
```

Comment:
I pulled down the feature branch to view the changes. I had to comment out the property of height at 100vh to be able see the whole page. It cut off the nav and header completely. Maybe it's just on mine?